GitUI/CommandsDialogs/SettingsDialog/SettingsTreeViewUserControl.cs

```diff
@@ -32,17 +33,18 @@ public SettingsTreeViewUserControl()
 
             treeView1.ImageList = new ImageList();
             treeView1.ImageList.ImageSize = DpiUtil.Scale(new Size(16, 16)); // Scale ImageSize and images scale automatically
+            treeView1.ImageList.Images.Add(Images.Blank);
             this.AdjustForDpiScaling();
         }
 
         /// <param name="asRoot">only one page can be set as the root page (for the GitExt and Plugin root node)</param>
-        public void AddSettingsPage(ISettingsPage page, SettingsPageReference parentPageReference, bool asRoot = false)
+        public void AddSettingsPage(ISettingsPage page, SettingsPageReference parentPageReference, Bitmap icon = null, bool asRoot = false)
```

Comment:
I would make `Bitmap icon` a mandatory parameter, the two (?) instances that don't have images can explicitly supply `null`

Reply:
Will be fixed